docs/UserGuide.md

```diff
@@ -38,30 +38,46 @@ StudyTracker is a **desktop app that lets you save your favourite study spots, o
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Command Syntax [incomplete]
+## Command Syntax
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
+**:information_source: Throughout the User Guide, you may see commands to enter into the bot. Here is how to read the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* The first word of every line *always* specifies the **command word**. Command words are single words. Commands may have aliases, which are shortened versions of the command word. <br>
+  e.g. `add` has the alias `new`.
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+* The subsequent blocks are known as **parameters**. Each parameter has a delimiter. <br>
+  e.g. the `NAME` parameter has a delimiter `n` and the `AMENITY` parameter has a delimiter `m`.
+
+* Square brackets are required for user input. Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n[NAME]*`, `NAME` is a parameter which can be used as `add n[COM1 Basement]`.
```

Comment:
We'll change from square brackets to the original format of `n/NAME*` for required inputs and `t/TAG` for the optional ones. Once this change is made, should be ok to merge!